components/post_view/post_list_virtualized/post_list_virtualized.jsx

```diff
@@ -510,30 +713,34 @@ class PostList extends React.PureComponent {
-                                        onScroll={this.onScroll}
-                                        onItemsRendered={this.onItemsRendered}
-                                        initScrollToIndex={this.initScrollToIndex}
-                                        canLoadMorePosts={this.props.actions.canLoadMorePosts}
-                                        skipResizeClass='col__reply'
-                                        innerRef={this.postListRef}
-                                        style={{...virtListStyles, ...dynamicListStyle}}
-                                        innerListStyle={postListStyle}
-                                        initRangeToRender={this.initRangeToRender}
-                                        loaderId={PostListRowListIds.OLDER_MESSAGES_LOADER}
-                                        correctScrollToBottom={this.props.atLatestPost}
-                                    >
-                                        {this.renderRow}
-                                    </DynamicSizeList>
+                                    <React.Fragment>
+                                        <div>{this.renderToasts(width)}</div>
+                                        <DynamicSizeList
+                                            ref={this.listRef}
+                                            height={height}
+                                            width={width}
+                                            className='post-list__dynamic'
+                                            itemCount={this.state.postListIds.length}
+                                            itemData={this.state.postListIds}
+                                            itemKey={this.itemKey}
+                                            overscanCountForward={OVERSCAN_COUNT_FORWARD}
+                                            overscanCountBackward={OVERSCAN_COUNT_BACKWARD}
+                                            onScroll={this.onScroll}
+                                            initScrollToIndex={this.initScrollToIndex}
+                                            canLoadMorePosts={this.props.actions.canLoadMorePosts}
+                                            skipResizeClass='col__reply'
+                                            innerRef={this.postListRef}
+                                            style={{...virtListStyles, ...dynamicListStyle}}
+                                            innerListStyle={postListStyle}
+                                            initRangeToRender={this.initRangeToRender}
+                                            loaderId={PostListRowListIds.OLDER_MESSAGES_LOADER}
+                                            correctScrollToBottom={this.props.atLatestPost}
+                                            onItemsRendered={this.onItemsRendered}
+                                            scrollTofailed={this.scrollTofailed}
```

Comment:
Can we fix the case on this?

Reply:
Unfortunately not. We wont be able to scroll to posts which are not mounted. It is something that is really hard to do so it easier to understand that a post is not mounted and use the index to try and figure out to position the UI

Reply:
I mean, can we fix the case of the name? It should be `scrollToFailed` instead of `scrollTofailed`

Reply:
Oops sorry got it.